Makefile

```diff
@@ -165,6 +165,8 @@ VDB_NAMESPACE?=default
 MINIMAL_VERTICA_IMG ?=
 # Name of the helm release that we will install/uninstall
 HELM_RELEASE_NAME?=vdb-op
+# Name of the cluster scope release that will be used for prometheus in the namespace scope release
+CLUSTER_SCOPE_RELEASE_NAME?=
```

Comment:
If there is only one cluster-wide operator, is there a way to fetch this name automatically?